easybuild/easyconfigs/b/Bowtie/Bowtie-1.1.2-foss-2015b.eb

```diff
@@ -0,0 +1,19 @@
+#Modified from existing version by:
+#Robert Schmidt <rjeschmi@gmail.com>
+#Ottawa Hospital Research Institute - Bioinformatics Team
+name = 'Bowtie'
+version = '1.1.2'
+
+homepage = 'http://bowtie-bio.sourceforge.net/index.shtml'
+description = """Bowtie is an ultrafast, memory-efficient short read aligner.
+It aligns short DNA sequences (reads) to the human genome.
+"""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+#toolchainopts = {'optarch': True, 'pic': True}
```

Comment:
Using `WITH_TBB` implies including `tbb` as a dependency, so it's a little bit more complicated...

There is a free version of TBB available though, so it's worth considering.

@nathanhaigh: can you clean up this line so we can get this merged?